roles/check-swlib/tasks/main.yml

```diff
@@ -16,13 +16,27 @@
 - name: check-swlib | Validate base software release
   shell: |
     set -o pipefail
-    {% for i in item.files %}
-      gsutil ls -L gs://{{ swlib_mount_src }}/{{ i }} | \
-        awk '/md5/ { print $3 }'
+    {% for f in item.files %}
+    if gsutil ls gs://{{ swlib_mount_src }}/{{ f.name }} >/dev/null 2>&1; then
+      f_name="{{ f.name }}"
+      md5sum="{{ f.md5sum }}"
+    else
+      f_name="{{ f.alt_name | default('x') }}"
```

Comment:
It might not be obvious to the reader what an "alt name" is;  could we add some descriptive text somewhere?